x-pack/plugins/security_solution/public/timelines/components/open_timeline/helpers.ts

```diff
@@ -435,18 +435,22 @@ export const dispatchUpdateTimeline =
     preventSettingQuery,
   }: UpdateTimeline): (() => void) =>
   () => {
-    if (!isEmpty(timeline.indexNames)) {
+    let _timeline = timeline;
```

Comment:
this comment is purely subjective so feel free to ignore it! While I understand why you name the variable `_timeline`, I feel like it's not super clear. I'm having a hard time finding a good name, but because the function's name is `dispatchUpdateTimeline` maybe we could replace `_timeline` by `updatedTimeline`?

Reply:
lol, well never mind 😆